### DISCUSSION TITLE
wrfout to netcdf

### DISCUSSION THREAD

**Discussion by rajkumar8581** [pydata/xarray#7286] [Q&A] [answered]
Hello all, I am trying to use xwrf to postprocess wrfout file and write to a cf compliant netcdf file using the following. Does nt work, can anyone please help?

import xarray as xr
import xwrf
from datetime import datetime as dt
ds= xr.open_dataset("wrfout_d01.nc").xwrf.postprocess()
ds.to_netcdf("test.nc")

**Comment by dcherian** [comment, marked as answer]
Please ask over at https://github.com/xarray-contrib/xwrf please
